bigbluebutton-html5/imports/ui/components/external-video-player/component.jsx

```diff
@@ -39,8 +44,23 @@ class VideoPlayer extends Component {
     window.addEventListener('resize', this.resizeListener);
   }
 
-  componentDidUpdate(nextProps) {
-    if (!nextProps.videoId) {
+  componentDidUpdate(prevProps) {
+    const { inEchoTest } = this.props;
+    const {
+      mutedByEchoTest,
+    } = this.state;
+
+    if (inEchoTest && !this.player.isMuted() && !mutedByEchoTest) {
+      this.player.mute();
+      this.changeState(true);
+    }
+
+    if (!inEchoTest && prevProps.inEchoTest && mutedByEchoTest) {
+      this.player.unMute();
+      this.changeState(false);
+    }
+
+    if (!prevProps.videoId) {
```

Comment:
The line deleted was ` if (!nextProps.videoId) {` please double check that replacing with ` if (!prevProps.videoId) {` yields the same result

Reply:
It's the same, `componentDidUpdate` returns the prevProps independent of variable name.

https://reactjs.org/docs/react-component.html#componentdidupdate